<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialogUI</class>
 <widget class="QDialog" name="InputDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="videoTypeButton">
       <property name="text">
        <string>下载视频</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="commentTypeButton">
       <property name="text">
        <string>下载评论</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="monitorButton">
       <property name="text">
        <string>监控直播</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="cleanButton">
       <property name="text">
        <string>清理</string>
       </property>
       <property name="icon">
        <iconset resource="crossdownloader.qrc">
         <normaloff>:/CrossDownloader/Resources/clean.png</normaloff>:/CrossDownloader/Resources/clean.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>支持输入视频链接、口令、用户页面、搜索链接、合辑</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>添加到:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>11</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="downloadPath"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="dirButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="confirmButton">
     <property name="styleSheet">
      <string notr="true">padding:8px</string>
     </property>
     <property name="text">
      <string>立即添加</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="crossdownloader.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cleanButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
